src/deidentifyPS315E.ts

```diff
@@ -390,6 +391,22 @@ export default function deidentifyPS315E({
             'notInRtnAdditionalIds',
             undefined,
           ]
+        } else if (
+          manualBlacklistSet.has(normalName) &&
+          vr !== 'UI' && // Don't blacklist UIDs - they have special handling
```

Comment:
reminder to remove this as we shouldn't put UIs in blacklist - instead, ensure UIs don't get flagged by scripts or blacklisted.